src/view/drag-handle/sensor/create-keyboard-sensor.js

```diff
@@ -18,8 +18,8 @@ type State = {|
 |}
 
 type ExecuteBasedOnDirection = {|
-  vertical: () => void,
-  horizontal: () => void,
+  vertical: Function,
+  horizontal: Function,
```

Comment:
Flow recommends [not using Function](https://flow.org/en/docs/types/functions/#toc-function-type), did you have issues with `() => void` or `() => mixed`?

Reply:
Some functions return a `AnimationFrameID`. I can move to () => mixed but i think just simply saying it is a `Function` is sufficient for our purposes here